### PR TITLE
add ftp download job managment

### PR DESCRIPTION
**Pull request by MarcAntoine-Arnaud**

### CODE REVIEW

File: assets/src/app/jobs/jobs.component.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      <mat-icon>compare_arrows</mat-icon>
+    </span>
+    <span class="title">
+      {{job.name | jobType}}
+    </span>
+    <span class="status">
+      {{job.status | jobStatus}}
+    </span>
+    <span class="date">
+      {{job.inserted_at | date: 'd LLL yy'}}
```
Comment (valnoel) [inline assets/src/app/jobs/jobs.component.html]: We could create a specific PipeTransform that would use Moment.js to handle date internationalization, and so display it in French in this case, don't you think?

File: assets/src/app/jobs/jobs.component.less [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      min-width: 80px;
+      flex: 1;
+    }
+    span.source.path {
+      justify-content: flex-end;
+    }
+  }
+
+  li.completed span.status {
+
```
Comment (valnoel) [inline assets/src/app/jobs/jobs.component.less]: Useless empty line!

File: assets/src/app/jobs/jobs.component.less [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    background-color: #71bd2c;
+  }
+  li.processing span.status {
+    background-color: #3965ac;
+    color: #fff;
+  }
+  li.error span.status {
+    background-color: #db280f;
```
Comment (valnoel) [inline assets/src/app/jobs/jobs.component.less]: Shouldn't we use Less variables to define common colors? So we could keep a common style into the GUI.

File: assets/src/app/services/job.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+
+
```
Comment (valnoel) [inline assets/src/app/services/job.ts]: Useless empty line

File: assets/src/app/jobs/jobs.component.html (hunk 1 above)
Comment (MarcAntoine-Arnaud) [reply assets/src/app/jobs/jobs.component.html]: A little overkill at this step I think.

**Pull request merged**